src/main/java/com/example/report/rabbitMQ/BenefitConsumer.java

```diff
@@ -0,0 +1,28 @@
+package com.example.report.rabbitMQ;
+
+import com.example.report.model.*;
+import com.example.report.repository.BenefitRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+import java.util.function.Consumer;
+
+
+@RequiredArgsConstructor
+@Component
+public class BenefitConsumer {
+
+    private final BenefitMapper benefitMapper;
+    private final BenefitRepository benefitRepository;
+
+
+    @Bean
+    public Consumer<BenefitDTOForRabbitMQ> benefitInput() {
+        return benefitDTOForRabbitMQ -> {
+            Benefit benefit = benefitMapper.dtoToEntity(benefitDTOForRabbitMQ);
+            benefitRepository.save(benefit);
+            System.out.println("MESSAGE RECEIVED: Benefit added in DB!");
```

Comment:
Let's replace `System.out.println` with some logging.
For example, we can add `Slf4j` annotation for this class and use `log.info()`